tests/callbacks/test_mlperf_callback.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import numpy as np
+import pytest
+from torch.utils.data import DataLoader
+
+from composer import Trainer
+from composer.callbacks import MLPerfCallback
+from tests.common import RandomClassificationDataset, SimpleModel
+
+logging.basicConfig(filename="/Users/hanlintang/composer/package_checker.log", level=logging.INFO)
```

Comment:
Pytest has built-in log capturing so I don't think we should need to configure this explicitly? We might need to adjust what packages / loglevels are being logged tho.

Reply:
The MLperf logging package uses `logging`, so these need to be configured. However, we should remove the hardcoded path there.